puffin_http/Cargo.toml

```diff
@@ -14,7 +14,10 @@ include = ["**/*.rs", "Cargo.toml", "README.md"]
 
 [dependencies]
 anyhow = "1.0"
-crossbeam-channel = "0.5"
+async-executor = "1.4.1"
+async-std = "1.12"
+flume = "0.10.14"
```

Comment:
flume seems like a nice change for crossbeams_channel

Reply:
Are you interested to change from crossbeams_channel to flume even without async ?

Reply:
I gave flume a go but the crossbeam benchmarks seems to indicate that crossbeam performed better than flume:

![plot](https://user-images.githubusercontent.com/19969910/198101760-063be846-c573-4832-a5d0-3b941d63f453.png)

And also their own benchmarks seemed to show that. So I am not very sure if its desirable. Maybe I am missing something tho

Reply:
Ok.
From what I have seen on flume crate page, most of the time crossbeam-channel is faster, but sometime flume do best.
It could be depend of principal usage in Puffin and whether the difference in performance is significant.
I have used flume in first place because crossbeam-channel don't manage async, and flume is presented as better than std (async ?) channel

Reply:
Yea makes sense to opt for that solution in this case. So its good to me using flume i.e. of crossbeam. Unless someone has an better idea lets opt for that.